Source/options.cpp

```diff
@@ -730,7 +731,7 @@ void OptionEntryResolution::CheckResolutionsAreInitialized() const
 	sizes.erase(std::unique(sizes.begin(), sizes.end()), sizes.end());
 
 	for (auto &size : sizes) {
-		resolutions.emplace_back(size, fmt::format("{}x{}", size.width, size.height));
+		resolutions.emplace_back(size, StrCat(size.width, "x", size.height));

```

Comment:
Can we make this a string cast/conversion operator on size? Could probably get rid of the ostream function for testing if it's allowed as an implicit conversion... but not sure if that'd cause issues elsewhere in the code.

Reply:
It is not obvious that the string conversion would produce this format. I think it's better to keep it explicit here.